app/dao/UserDao.scala

```diff
@@ -55,14 +58,23 @@ class UserDao @Inject()(repo: UserRepo)(implicit ec: ExecutionContext)
   def auth(user: common.User): Future[Option[common.User]] =
     auth(user.getName, user.getPassword)
 
+  def getUser(id: Long): Future[Option[common.User]] = {
+    //    TODO:rework this!
+    logger.info(s"Obtaining user with id $id")
+    repo.find(id).map { f =>
+      f.map { u =>
```

Comment:
`f`
`u`
:panic:

Reply:
yeaah, there's `todo` right next to it, not sure how to make it better =(

Reply:
ANYTHING but this :C